policy/README.md

```diff
@@ -1,21 +1,23 @@
-# CNCF Security SIG Policy Team Introduction
+# CNCF SIG Security Policy Team Introduction
```

Comment:
Sentence case should be "CNCF SIG Security policy team introduction"

Reply:
done